src/main/java/service/OptionCli.java

```diff
@@ -0,0 +1,113 @@
+package service;
+
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.cli.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Slf4j
+@Data
+public class OptionCli {
+    private String[] arguments;
+    private String dataTime;
+    private CommandLineParser parser;
+    private Options options;
+    private CommandLine cmd;
+
+    public OptionCli(String[] arguments) {
+        this.arguments = arguments;
+        this.parser = new DefaultParser();
+        this.options = optionsBuild();
+        try {
+            this.cmd = parser.parse(this.options, this.arguments);
+        } catch (ParseException e) {
+            log.info("Invalid values ​​for parse... \n\tFor help, type -h or --help");
+        }
+    }
+
+    private Options optionsBuild() {
+        this.options = new Options();
```

Comment:
jeśli w 22 linijce robisz `this.options = optionsBuild();` to tutaj zadeklaruj `options` jako zmienna lokalną którą na końcu zwrócisz